spark_log_parser/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Tools for providing Spark event log"""
+
+__version__ = "0.0.1"
```

Comment:
Are we responsible for manually updating this every time a commit is made to main?

Reply:
Every time a commit is made that we want to expose to our other projects, yes.